src/collections/_documentation/platforms/cocoa/dsym.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Uploading Debug Symbols'
+---
+
+A dSYM upload is required for Sentry to symoblicate your crash logs. The symoblication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. The dSYM file can be uploaded through [sentry-cli](https://github.com/getsentry/sentry-cli) or through a [Fastlane](https://fastlane.tools/) action.
+
+## With Bitcode {#dsym-with-bitcode}
+
+If [Bitcode](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AppThinning/AppThinning.html#//apple_ref/doc/uid/TP40012582-CH35-SW2) **is enabled** in your project, you will have to upload the dSYM to Sentry **after** it has finished processing in iTunesConnect. We also recommend using the latest Xcode version for submitting your build. The dSYM can be downloaded in three ways.
+
+### Using Fastlane
+
+Use the [Fastlane](https://github.com/fastlane/fastlane) action, _download_dsyms_, to download the dSYMs from iTunesConnect and upload to Sentry. The dSYM won’t be generated until **after** the app is done processing on iTunesConnect so this should be run in its own lane.
+
+```ruby
+lane :upload_symbols do
+  download_dsyms # this is the important part
+  upload_symbols_to_sentry(
+    auth_token: 'YOUR_AUTH_TOKEN',
+    org_slug: '___ORG_NAME___',
+    project_slug: '___PROJECT_NAME___',
+  )
+end
+```
+
+{% capture __alert_content -%}
+By default fastlane will connect to sentry.io. For on-prem you need to provide the _api_host_ parameter to instruct the tool to connect to your server:
+
+```
+api_host: 'https://mysentry.invalid/'
+```
+{%- endcapture -%}
+{%- include components/alert.html
+  title="On Prem"
+  content=__alert_content
+%}
+
+### Using `sentry-cli`
+
+There are two ways to download the dSYM from iTunesConnect. After that, you can upload the dSYM using [sentry-cli](https://github.com/getsentry/sentry-cli/releases).
```

Comment:
I find this a bit confusing. We mention two ways to download, but I'm not sure what the methods are? 

For structuring, suggest: "Download the dSYM from iTunesConnect using either ... or ..."